pkg/client/client.go

```diff
@@ -86,6 +91,70 @@ func (c *ImmuClient) Get(keyReader io.Reader) (*schema.Item, error) {
 	return result, err
 }
 
+// VerifiedItem ...
+type VerifiedItem struct {
+	Key      []byte `json:"key"`
+	Value    []byte `json:"value"`
+	Index    uint64 `json:"index"`
+	Verified bool   `json:"verified"`
+}
+
+// SafeGet ...
+func (c *ImmuClient) SafeGet(keyReader io.Reader) (*VerifiedItem, error) {
+	start := time.Now()
+	if !c.isConnected() {
+		return nil, ErrNotConnected
+	}
+	key, err := ioutil.ReadAll(keyReader)
+	if err != nil {
+		return nil, err
+	}
+
+	rs := NewRootService(c.serviceClient, cache.NewFileCache())
```

Comment:
Can you use dependency injection for RootService?

Reply:
@mmeloni Please see my last commit : https://github.com/codenotary/immudb/pull/40/commits/fb49cb630987c7e5178f8b83ee146fdf6cedb461

and let me know what you think. Thanks.

Reply:
ok!